src/ontodia/diagram/view.ts

```diff
@@ -432,9 +432,122 @@ export class DiagramView extends Backbone.Model {
                 end: this.createLinkMarker(linkTypeId, false, style.markerTarget),
             };
         }
+        style.router = this.getLinkRouter();
```

Comment:
It would be better to provide default router as a fallback in `LinkView` if it isn't specified in user-provided `LinkStyle`.